cloudformation/master-server-substack.cfn.yaml

```diff
@@ -341,6 +367,36 @@ Resources:
         NetworkInterfaces:
           - NetworkInterfaceId: !Ref 'MasterENI'
             DeviceIndex: 0
+          - !If
+            - UseNIC1
+            - DeviceIndex: 1
+              NetworkCardIndex: 1
+              InterfaceType: !Ref 'AWS::NoValue'
+              Groups: !Ref 'MasterSecurityGroups'
```

Comment:
Should we pass Groups, SubnetId etc to the first network interface?

Reply:
The first one is the MasterENI which has already all them assigned